indicators/tuya-redirect-4ewqnc.yml

```diff
@@ -0,0 +1,38 @@
+title: Tuya Redirect 4eWQNc
+description: |
+    Detects a phishing landing page for the Colombian bank and credit card issuer Tuya.
+    This was found as a result of this kit being deployed on Replit.
+
+
+references:
+    - https://www.tuya.com.co/
+    - https://urlscan.io/result/026d1d48-ba58-477d-a168-0581ef8b9ebc/
```

Comment:
This reference seems to be broken, could you please provide another one that matches this rule?

Reply:
I believe I have found another valid reference you can use `https://urlscan.io/result/630122d4-6b1b-4b56-bb13-6b6a1f28af4f/`